R/ggsave_optimized.R

```diff
@@ -0,0 +1,75 @@
+#' Save Optimized Image Formats
```

Comment:
Not clear what "optimized" means here. By reading the code it looks like it's aiming to minimize output file size, but that's not obvious from the description.